share/functions/funced.fish

```diff
@@ -105,9 +105,7 @@ function funced --description 'Edit function definition'
                 echo # add a line between the parse error and the prompt
                 set -l repeat
                 set -l prompt (_ 'Edit the file again\? [Y/n]')
-                while test -z "$repeat"
-                    read -p "echo $prompt\  " repeat
-                end
+                read -p "echo $prompt\  " repeat
```

Comment:
This is a step in the right direction. The problem is that it accepts any random string, as well as the empty string, as being an answer in the affirmative if it isn't some variation of "no". That's dangerous. Obviously this was an issue  in the original version and not something you introduced. Nonetheless, since you're touching this would you mind also explicitly testing for one of the acceptable affirmative answers (which include the empty string)?